### PR TITLE
Add highlight class for endpoints and connectors

### PR DESCRIPTION
**Pull request by SteveBarnett**

### CODE REVIEW

File: go/base/static/css/conversations.less [hunk 1]
```diff
@@ -259,6 +260,18 @@
...
+    .highlighted {
+      &.endpoint {
+        background-color: @color-highlighted;
+      }
+      path, circle {
+        stroke: @color-highlighted;
+      }
+      circle, [pointer-events="all"] {
+        fill: @color-highlighted;
+      }
```
Comment (justinvdm) [inline go/base/static/css/conversations.less:273]: Could you explain why we need `[pointer-events="all"]` here?

**Comment by justinvdm** [comment]
@SteveBarnett Sorry, just realised that this probably isn't ready for review yet.

File: go/base/static/css/conversations.less (hunk 1 above)
Comment (SteveBarnett) [inline go/base/static/css/conversations.less:273]: I need to target the arrowheads, and it seems only they have `[pointer-events="all"]`.
¯\_(ツ)_/¯

Comment (justinvdm) [inline go/base/static/css/conversations.less:273]: Ah, I see, cool thanks :)

**Comment by justinvdm** [comment]
@SteveBarnett I think we can probably close this now, right?

**Comment by SteveBarnett** [comment]
@justinvdm Yeah. Not going to finish it off.

**Pull request closed without merging**